README.md

```diff
@@ -129,22 +129,29 @@ add_filter( 'wp_sentry_dsn', 'customize_sentry_dsn' );
 
 #### `wp_sentry_options` (array)
 
-You can use this filter to customize the Sentry options used to initialize the PHP tracker.
+You can use this filter to customize the [Sentry options](https://docs.sentry.io/error-reporting/configuration/) used to initialize the PHP tracker.
```

Comment:
You said it right here 👍 this is for the PHP tracker _only_.

There is a different filter `wp_sentry_public_options` for the JS SDK you should change instead of this one.